frontend/src/components/Credentials/GCredentialDetailsItemContent.vue

```diff
@@ -71,7 +71,10 @@ export default {
         ]
       }
       if (this.credential?.kind === 'Secret') {
-        return this.getCredentialDetails(this.credential)
+        const details = this.getCredentialDetails(this.credential)
+        if (details) {
+          return details
+        }
       }
       if (this.credential?.kind === 'WorkloadIdentity') {
```

Comment:
shouldn't this be rather an else-if / switch statement?
(I know this code was not touched, but I thought I gave this feedback already in the previous CredentialBinding support PR)?